### PR TITLE
Update qwen2.py - Use `gpt_config` values

### PR DESCRIPTION
**Pull request by SamuelWN**
Implement TranslatorConfig; various changes throughout to accommodate  (read in config variables, format `_CHAT_SAMPLE` as `Dict[str, List[str]]`, etc.)

Tweaked logging. 

Added `attention_mask`, padding, etc. to `tokenizer` call.

### CODE REVIEW

**Comment by SamuelWN** [comment]
I encountered some instances where parsing the identifiers seems to fail. e.g.

```
Original:
<|1|>別の日
<|2|>おとなしくしろ！
<|3|>おっとさわぐなよ…おとなしくしていれば悪いようにはしねえ
<|4|>な…！
<|5|>え⁉
<|6|>なんかおだやかじゃないな…
<|7|>どっきり⁉

[Qwen2BigTranslator] -- Qwen2 Response --
<|1|>Another day,
<|2|>Be quiet!  
<|3|>Hold your horses... If you behave yourself, you won't get into trouble.
<|4|>Geez...!
<|5|>What?!  
<|6|>You're not acting calm at all...
<|6|>A surprise?!
[Qwen2BigTranslator] 0: 別の日 => <|1|
[Qwen2BigTranslator] 1: おとなしくしろ！ => Another day, <|2|
[Qwen2BigTranslator] 2: 殺人犯が生徒会室にろう城するとかどうだ => Be quiet! <|3|
[Qwen2BigTranslator] 3: な…！ => Hold your horses... If you behave yourself, you won't get into trouble. <|4|
[Qwen2BigTranslator] 7: え⁉ => Geez...! <|5|
[Qwen2BigTranslator] 8: なんかおだやかじゃないな… => What?! <|6|
[Qwen2BigTranslator] 9: どっきり⁉ => You're not acting calm at all... <|7|

```

I'll investigate and re-submit when resolved.
